### PR TITLE
Task2 - download contributors avatars

### PR DESCRIPTION
**Pull request by ViraBakhovska**

### CODE REVIEW

File: Task3/download_contributors_avatars.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+﻿"""This module creates {username}/{project} folders
+and download the avatar picture files with names
+that match GitHub contributor’s login"""
+import argparse
+import json
+import os
+import requests
+
+USER_NAME = 'ViraBakhovska'
```
Comment (trnvsk) [inline Task3/download_contributors_avatars.py]: Please add two more arguments to the argparser for your username/token.

File: Task3/download_contributors_avatars.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+folder_name = "{}{}{}".format(args.user, os.sep, args.project)
+os.makedirs(folder_name, exist_ok=True)
+
+
+for contributor in contributors_avatar_dict:
+    contributor_response = gh_session.get(contributor['avatar_url'], stream=True)
+    file_extension = contributor_response.headers['Content-Type'].split('/')[1]
+
+    if file_extension in ['jpeg', 'png']:
+        image_name = f'{folder_name}\\{contributor["login"]}.{file_extension}'
```
Comment (trnvsk) [inline Task3/download_contributors_avatars.py]: Please check os.path.sep instead of hardcoding backslash in the path.

**Review by trnvsk — CHANGES_REQUESTED** [review]
Overall looks good, there are two comments.

File: Task3/download_contributors_avatars.py (hunk 1 above)
Comment (ViraBakhovska) [reply Task3/download_contributors_avatars.py]: added arguments for username/token.

File: Task3/download_contributors_avatars.py (hunk 2 above)
Comment (ViraBakhovska) [reply Task3/download_contributors_avatars.py]: fixed path

**Review by trnvsk — APPROVED** [review]
Looks good

**Pull request merged**